include/constants/battle.h

```diff
@@ -19,4 +19,22 @@
 #define BATTLE_TYPE_AI_PARTNER      (BATTLE_TYPE_DOUBLES | BATTLE_TYPE_2vs2 | BATTLE_TYPE_AI)
 #define BATTLE_TYPE_SPECIAL_WILD    (BATTLE_TYPE_SAFARI | BATTLE_TYPE_ROAMER | BATTLE_TYPE_PAL_PARK | BATTLE_TYPE_CATCH_TUTORIAL)
 
+#define MAX_LINK_BATTLERS   4
+#define MAX_BATTLERS        4
+#define MAX_BATTLE_SIDES    2
```

Comment:
I would go for `NUM_` here.

Reply:
I like `NUM_BATTLE_SIDES`, but the rest are actual maximums, since 1v1 battles exist.

Reply:
I see, I didn't think of that. In that case, keep it as `MAX_`.

Reply:
I'm keeping `NUM_BATTLE_SIDES`; we agreed in the `pokeheartgold` issue that `NUM` is useful for constant cardinalities, and there are *always* 2 sides in a battle. But there might be 2-4 active battlers in a given battle, so that one is a `MAX`.